CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## v4.4.0 - 2023-10-11
+
+- Add `localtime.strftime` [#100](https://github.com/octoenergy/xocto/pull/100)
+- Add `RangeSet.is_(left|right)_finite` [#96](https://github.com/octoenergy/xocto/pull/96)
```

Comment:
Looks like this change didn't get released yet https://github.com/octoenergy/xocto/pull/96

Reply:
> Looks like this change didn't get released yet https://github.com/octoenergy/xocto/pull/96

I'll get a release out.